### ISSUE TITLE
Exception Maximum call stack size exceeded when trying to connect with Alexa

### ISSUE THREAD

**Issue by Garfonso** [ioBroker/ioBroker.cloud#101]
Hi,

I can not connect the cloud adapter to the alexa iobroker skill. I always get a maximum call stack size exceeded exception and the adapter restarts (see log). I did uncheck all enums in config right now and deleted most devices. But that did not seem to help. Any pointers? It seems it runs into an endless loop somewhere? 

[cloud.log](https://github.com/ioBroker/ioBroker.cloud/files/2399885/cloud.log)

**Comment by GermanBluefox** [comment]
Which version of cloud Adapter do you have?

**Comment by Garfonso** [comment]
Version 2.6.2

**Comment by Garfonso** [comment]
Hi,
did you change anything anywhere? Today I tried again and it worked. 

One thing I did change: I replaced a not properly configured KNX instance with a properly working one. 

Thanks anyway.

**Comment by Garfonso** [comment]
Hi,

today I update this ticket, because I have more information.
Connection to alexa was not the culprit, but the only thing that I noticed back then, because I did not use the remote connection.
I now updated to pro, because I like iobroker and also wanted to try the full remote access.
And I recognized that my cloud adapter is crashing again as soon as I try to access the remote access part of iobroker.pro or iobroker.net page.

I thought a bit, why it might have worked for connecting alexa once and I remembered that I had disabled ssl for a short time back then. So I disabled ssl again for web.0 and remote access works.

I can now reproduce this: If you enabled SSL for the web instance and try any kind of remote access via iobroker.net or iobroker.pro (i.e. log into the profile and click on the top most button), the cloud adapter crashes with the errors I posted above.

Some more information:
I added my own let's encrypt ssl certificates (which I generate on my server for my apache instance independently of iobroker) in ioBroker and used them. They work fine for local and direct (i.e. port forwarding) web access. 
One might argue that using SSL and the remote access via iobroker.net/pro is not necessary at all. But it would at least be helpful to add a note in the cloud-Adapter settings page and maybe catch the exceptions and print a more informative message in the log file.
